tests/utils/tab.spec.js

```diff
@@ -271,5 +271,23 @@ describe('tabUtils', () => {
       tabUtils.sendMessage(tab1, 'hello', true);
       expect(tab1.ref[0].postMessage).toHaveBeenCalled();
     });
+
+    it('should send message to the all ref windows existing', () => {
```

Comment:
test case above it can be removed now as it does the same thing but for target.ref[0]